misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestCmd.java

```diff
@@ -161,6 +175,17 @@ public void execute() {
         Runtime.getRuntime().exit(0);
     }
 
+    /**
+     * Identifies if a directory given is a ballerina project or not.
+     *
+     * @param projectPath project path.
+     * @return true if its a project else false.
+     */
+    private boolean isNotABallerinaProject(Path projectPath) {
```

Comment:
Cant we do with isBallerinaProject instead of Not ?

Reply:
Fixed with e5d6138. Thanks :)